tools/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.21
+FROM golang:1.22.5
```

Comment:
It looks like this change breaks the linter for some weird reason... IMO this is a good example that this specific file should probably NOT be checked and be put in an exclusion/exception list.
Otherwise bumping the Golang version becomes an exercise of fighting with the linter...

Reply:
I'll add an exceptions list.